mobile/application-manager-base.ts

```diff
@@ -44,13 +42,7 @@ export abstract class ApplicationManagerBase extends EventEmitter implements Mob
 					let newAppIdentifiers = _.difference(currentlyInstalledAppIdentifiers, previouslyInstalledAppIdentifiers);
 					let removedAppIdentifiers = _.difference(previouslyInstalledAppIdentifiers, currentlyInstalledAppIdentifiers);
 
-					if(newAppIdentifiers.length || removedAppIdentifiers.length) {
-						this.applicationsLiveSyncStatus = _.reject(this.applicationsLiveSyncStatus,
-															(app: Mobile.IApplicationLiveSyncStatus) => _.contains(removedAppIdentifiers, app.applicationIdentifier));
-						this.getApplicationsLiveSyncSupportedStatus(newAppIdentifiers).wait();
-					}
-
-					_.each(newAppIdentifiers, application => this.emit("applicationInstalled", application, {isLiveSyncSupported: this.isLiveSyncSupported(application).wait()} ));
+					_.each(newAppIdentifiers, application => this.emit("applicationInstalled", application));
```

Comment:
I'd rather you renamed the captured `application` argument to something like `applicationIdentifier` or `appIdentifier`